builtin/v13/check.go

```diff
@@ -288,6 +288,67 @@ func CheckDealStatesAgainstSectors(acc *builtin.MessageAccumulator, minerSummari
 		acc.Require(deal.SlashEpoch <= sectorDeal.SectorExpiration,
 			"deal state slashed at %d after sector expiration %d for miner %v",
 			deal.SlashEpoch, sectorDeal.SectorExpiration, deal.Provider)
+
+		acc.Require(deal.SectorNumber == sectorDeal.SectorNumber,
```

Comment:
You probably need to exclude `deal.SectorNumber == 0` here (ideally it would be `(deal.SectorNumber == 0 && deal.LastUpdatedEpoch > currEpoch)`).

Reply:
Added the condition